### PR TITLE
Fix #272042: saved preferences override command line options

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
For master, here at least do the input check and update the usage message.

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -6006,7 +6006,10 @@ int main(int argc, char* av[])
             QString temp = parser.value("r");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            preferences.setTemporaryPreference(PREF_EXPORT_PNG_RESOLUTION, temp.toDouble());
+            bool ok = false;
+            double res = temp.toDouble(&ok);
+            if (ok)
+                  preferences.setTemporaryPreference(PREF_EXPORT_PNG_RESOLUTION, res);
```
Comment (lasconic) [inline mscore/musescore.cpp]: Well, since we are at it, let's add a non fatal (?) error message?

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp]: Something simple as this?
<cpp>
            else
                  fprintf(stderr, "PNG resolution not recognized, using default settings from preferences instead\n");
</cpp>

Comment (lasconic) [reply mscore/musescore.cpp]: Yes.

Comment (lasconic) [reply mscore/musescore.cpp]: quoting the value that we parsed ?

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp]: Yes.

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp]: Will do the same for the 2.3 PR

Comment (Jojo-Schmitz) [reply mscore/musescore.cpp]: Also putting out the usage message, using `parser.helpText();`? Before or after the error message?

Comment (lasconic) [reply mscore/musescore.cpp]: maybe too much no ?

**Pull request merged**